### DISCUSSION TITLE
Turtle disk loading side

### DISCUSSION THREAD

**Discussion by PhobiaBrixia** [cc-tweaked/CC-Tweaked#617] [Q&A]
I've tried to replicate a program made by Michael Reeves, only to find out that in this version you cannot do a disk drive startup script loading it from the right side as with old computercraft (where people would choose to put a pickaxe on the right side as to allow the turtle to load the disk), but apparently only from the slit side. Is it intended? And also there's the problem with turtle stacking but there's already another thread open about it.

**Comment by Lupus590** [comment]
When a turtle looks for a peripheral on one of its sides it first checks what's attached to it and attempts to use those. If what it has got attached is not a peripheral or it has nothing attached on that side then it will instead attempt to use the block on that side as a peripheral instead. Tools do not count as peripherals, and so wrapping that side will attempt to wrap the block.

**Comment by SquidDev** [comment]
I'm a little confused about what you're asking. Could you post a screenshot of what you'd expect to work, as well as Minecraft and mod versions? Just tested a disk drive on mainline 1.15.2, and the disk appears to be mounted as one would expect: 

![Turtle with a disk on the right](https://user-images.githubusercontent.com/4346137/102994952-fe6c3200-4517-11eb-9eaf-0a374b36219b.png)

**Reply by PhobiaBrixia** [reply]
I'm playing on 1.16.4, the program outputs turtles from a main "mother" turtle, with the disk drive on the pickaxe side of the outgoing turtle, just like your picture, but in my case it doesn't work.
